RxExample/RxExample/Examples/WikipediaImageSearch/Views/WikipediaSearchViewController.swift

```diff
@@ -44,9 +44,13 @@ class WikipediaSearchViewController: ViewController {
 
         let API = DefaultWikipediaAPI.sharedAPI
 
-        searchBar.rx_text
-            .asDriver()
+        let searchText = searchBar.rx_text.asDriver()
             .throttle(0.3)
+        let searchTapped = searchBar.rx_searchButtonClicked.asDriver()
+            .flatMap { $0.text.flatMap { Driver.just($0) } ?? Driver.empty() }
```

Comment:
Why do we need search tapped event here, shouldn't `searchText` already contain current search text value?